proto/stored_invocation.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package stored_invocation;
```

Comment:
do we need a separate file? we already have a lot of custom messages in `invocation.proto`, could we add add this there?

Reply:
invocation.proto is used by the client. My main consideration here is to not expose the stored invocation proto to the client.

Reply:
You can leave it here if you feel strongly about it, but I don't see why it matters that the client happens to use the messages in invocation.proto. Adding the message there does not affect the client in any way.